src/plutarch-onchain-lib/lib/Plutarch/Core/List.hs

```diff
@@ -0,0 +1,155 @@
+{-# LANGUAGE QualifiedDo #-}
+module Plutarch.Core.List (
+  pdropFast,
+  pbuiltinListLengthFast,
+  penforceNSpendRedeemers,
+  phasNSetBits,
+  emptyByteArray
+) where
```

Comment:
https://github.com/input-output-hk/wsc-poc/blob/765832d8b59c3dd82da84a996adfb9a41ff1980c/src/lib/Plutarch/Extra/Record.hs

This is missing a few functions from the above.

The following are missing:
```haskell
  pisUniqueSet,
  phasNUniqueElements,
  ```